src/common/logging.cpp

```diff
@@ -53,6 +53,7 @@ static otbrLogLevel sLevel            = OTBR_LOG_INFO;
 static const char   sLevelString[][8] = {
       "[EMERG]", "[ALERT]", "[CRIT]", "[ERR ]", "[WARN]", "[NOTE]", "[INFO]", "[DEBG]",
 };
+static bool syslogDisabled = false;
```

Comment:
To be consistent with the code convention:
```suggestion
static bool sSyslogDisabled = false;
```

Add `s` prefix to standard for `static` variable.